### DISCUSSION TITLE
🔥🔥🔥欢迎讨论、闲聊、提问题~

### DISCUSSION THREAD

**Discussion by daGaiGuanYu** [ppz-pro/ppz.vscode#41] [Announcements]
这里是 PPZ.vscode 的讨论区，欢迎光临！

**Comment by kinfuy** [comment]
以后，会有切换到ts+vue的想法吗

**Reply by daGaiGuanYu** [reply]
现在已经在用 vue 了，大部分页面是用 vue 写的

ts 的话，因为有 esbuild 的存在，所以如果有需要的话，随时也可以使用  
但是目前还没觉得有用 ts 的必要

**Reply by kinfuy** [reply]
哇，看见一种新（小白认知里）的方式使用vue，感觉挺有趣的😄

**Reply by daGaiGuanYu** [reply]
回归原始的方式~

**Comment by kinfuy** [comment]
ts和用不用esbuild有什么关系吗

**Reply by daGaiGuanYu** [reply]
有 esbuild，混合语言会流畅些

**Comment by yyt6801** [comment]
催更呀！  oracle  DB2 也支持起来

**Reply by daGaiGuanYu** [reply]
最近太懒了……😢
